android/app/src/main/java/com/ptokenssentinelandroidapp/database/DatabaseWiring.java

```diff
@@ -238,22 +234,20 @@ public void endTransaction() throws DatabaseException {
             START_DB_TX_IN_PROGRESS = false;
             Log.v(TAG, "keys written, db tx ended successfully");
         }
-        /*
         try {
             if (writeSignedStateHashEnabled) {
-                writeSignedStateHash();    
+                writeSignedStateHash();
             } else {
-                Log.w(TAG, "skipping state hash writing...");                        
+                Log.w(TAG, "skipping state hash writing...");
             }
         } catch (DatabaseException e) {
             Log.e(TAG, "failed to write the state hash", e);
         } finally {
             START_DB_TX_IN_PROGRESS = false;
```

Comment:
We can't actually reach here without this already being false, so this is redundant. (See line #234)

Reply:
good catch